tools/ctc_segmentation/scripts/run_ctc_segmentation.py

```diff
@@ -116,10 +116,9 @@
 
         original_duration = len(signal) / sample_rate
         logging.debug(f'Duration: {original_duration}s, file_name: {path_audio}')
-        log_probs = asr_model.transcribe(paths2audio_files=[str(path_audio)], batch_size=1, logprobs=True)[0].cpu()
+        log_probs = asr_model.transcribe(paths2audio_files=[str(path_audio)], batch_size=1, logprobs=True)[0]
 
         # move blank values to the first column
-        log_probs = np.squeeze(log_probs, axis=0)
```

Comment:
?

Reply:
Now that it is a numpy array, the results do not have the extra 1 dimension and no need to apply squeeze.